utils/vara-ui/src/components/button/button.tsx

```diff
@@ -52,7 +53,7 @@ const Button = forwardRef<HTMLButtonElement, Props>((props, ref) => {
       className={cx(
         styles.button,
         styles[color],
-        color !== 'transparent' && styles[size],
+        styles[size],
```

Comment:
We can't remove transparent button without size, it can break UI in many places.